src/scenes/title/CreditsScene.ts

```diff
@@ -1,66 +1,59 @@
 import { Input, Scene } from "phaser";
-import { BackgroundImage } from "../components/BackgroundImage";
-import { CreditsText } from "../components/CreditsText";
-import { ITitleSceneInitData, TitleScene } from "./TitleScene";
+import { CreditsText } from "../../components/CreditsText";
 
 const cfg = { fadeOut: 800, timeOut: 320 };
 
 export class CreditsScene extends Scene {
-    private timers: number[] = [];
     private shuttingDown = false;
 
     constructor(key = "CreditsScene") {
         super(key);
     }
 
     public create() {
-        new BackgroundImage(this, "title");
         this.addCredits();
         this.stopEventPropagationToOtherScenes();
         this.input.on("pointerup", () => {
             this.shuttingDown = true;
             this.handleCreditEnd();
         });
+        this.events.on("wake", () => {
+            console.log("wake");
```

Comment:
Needs to be deleted in teh feature. As discussed, we will delete it in the future.